docs/_pages/quick-start-guide.md

```diff
@@ -13,6 +13,20 @@ Simulato is a model based testing tool to help provide full and variable coverag
 * [Selenium Webdriver](https://www.seleniumhq.org/docs/03_webdriver.jsp)
 * [Chai Assertions](http://www.chaijs.com/api/assert/)
 
+# Prerequisites
+* Node.js version 8+
+	* Download here: https://nodejs.org/en/
+* Chrome Browser installed
+* Chromedriver installed
+	* Download here: https://sites.google.com/a/chromium.org/chromedriver/downloads
+	* If on a mac
+		* You can `brew install chromedriver`
```

Comment:
I prefer not to use `brew` to manage dependencies and use package.json to manage project dependencies.